dynamic_programming/iterating_through_submasks.py

```diff
@@ -0,0 +1,59 @@
+"""
+Author : Syed Faizan (3rd Year Student IIIT Pune) 
+github : faizan2700
+You are given a bitmask m and you want to efficiently iterate through all of
+its submasks. The mask s is submask of m if only bits that were included in
+bitmask are set
+"""
+
+
+def list_of_submasks(mask) -> list:
```

Comment:
```python
from typing import List


def list_of_submasks(mask: int) -> List[int]:
```

Reply:
it is showing "Merging can be performed automatically once the requested changes are addressed." with red mark what does that mean ? have I not properly implemented changes?

Reply:
You have been asked to change the code so that it is compliant. Once your changes are approved they can be merged.

Reply:
@onlinejudge95  But I have made all the requested changes. 1. add the tests for mask<=0  2. change the line submask = ( submask - 1 ) & mask 3. change the line {def list_of_submasks(mask) -> list} to {def list_of_submasks(mask: int) -> List[int] }. is there any particular change I have to make to remove that red mark line of automatic merging?

Reply:
Thank you for your time

Reply:
Yup I will add my review now, was busy in work so couldn't do it just now